### PR TITLE
[WAF] Move FAQ from FW rules, update Security Level

### PR DESCRIPTION
**Pull request by pedrosousa**
Addresses:
- PCX-8934
- PCX-8615

### CODE REVIEW

File: content/waf/troubleshooting/faq.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
+---
+title: FAQ
+pcx_content_type: faq
+weight: 10
+---
+
+# WAF FAQ
```
Comment (pedrosousa) [inline content/waf/troubleshooting/faq.md:7]: Has essentially the same content as the `/content/firewall/known-issues-and-faq/index.md` file I deleted, with a few adaptations.

File: content/waf/tools/security-level.md [hunk 2]
```diff
@@ -6,36 +6,54 @@ title: Security Level
...
-| I’m Under Attack! | N/A | Only for use if your website is currently under a DDoS attack. |
+| Security Level                  | Threat score range | Description                                                                        |
+|---------------------------------|--------------------|------------------------------------------------------------------------------------|
+| Off (Enterprise<br>customers only) | _N/A_           | Does not challenge IP addresses.                                                   |
+| Essentially off                 | 50–100             | Only challenges IP addresses with the worst reputation.                            |
+| Low                             | 25–100             | Challenges only threatening visitors.                                              |
+| Medium                          | 15–100             | Challenges both threatening and moderately threatening visitors.                   |
+| High                            |  0–100             | Challenges all visitors that exhibit threatening behavior within the last 14 days. |
```
Comment (marciocloudflare) [inline content/waf/tools/security-level.md]:
```suggestion
| High                            |  0–100             | Challenges all visitors that exhibited threatening behavior within the last 14 days. |
```

File: content/waf/tools/security-level.md [hunk 3]
```diff
@@ -44,10 +62,10 @@ To update the Security Level for your entire zone:
 
 ### Update selectively
 
-If you wanted to set the Security level more selectively:
+To set the Security level more selectively, do one of the following:
```
Comment (marciocloudflare) [inline content/waf/tools/security-level.md]:
```suggestion
To set the security level more selectively, do one of the following:
```

Comment (marciocloudflare) [reply content/waf/tools/security-level.md]: You lowered the case for `security` above

**Review by marciocloudflare — APPROVED** [review]

File: content/waf/tools/security-level.md (hunk 3 above)
Comment (pedrosousa) [reply content/waf/tools/security-level.md]: Yes, I did... and should've done the same here. Nice catch!

**Pull request merged**